src/components/Blog/Blog.styles.tsx

```diff
@@ -1,3 +1,39 @@
+//components
 import styled from 'styled-components';
 
-export const TitlePost = styled.h3``;
+//interface
+import { ImageProps } from '@/interface/interface';
+
+//theme
+import theme from '@/theme/theme';
+
+export const ImagePost = styled.img<ImageProps>`
+    max-width: 40%;
+    min-width: 250px;
+    margin: 1rem;
+    top: 0;
+    left: 0;
+    float: right;
```

Comment:
Why float?

Reply:
po to, żeby zdjęcie było otoczone tekstem - nie znalazłam rozwiązania z flexboxem do tego. z tego co czytałam flexbox nie poradzi sobie z tym, żeby teks znajdował się obok i pod zdjęciem.